### PR TITLE
*: set errcheck exclude func

### PR DESCRIPTION
**Pull request by okJiang**
### What problem does this PR solve?

Issue Number: Close https://github.com/tikv/pd/issues/1919

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Code changes

- Has the configuration change
- Has HTTP API interfaces changed (Don't forget to [add the declarative for the new API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by HuSharp — APPROVED** [review]

File: pkg/schedule/schedulers/balance_leader.go [hunk 1]
```diff
@@ -86,10 +86,12 @@ func (conf *balanceLeaderSchedulerConfig) Update(data []byte) (int, any) {
 	newConfig, _ := json.Marshal(conf)
 	if !bytes.Equal(oldConfig, newConfig) {
 		if !conf.validateLocked() {
-			json.Unmarshal(oldConfig, conf)
+			_ = json.Unmarshal(oldConfig, conf)
```
Comment (rleungx) [inline pkg/schedule/schedulers/balance_leader.go:89]: Should we handle the error?

Comment (okJiang) [reply pkg/schedule/schedulers/balance_leader.go:89]: I can add a log for this. It seems like nothing else can be done.

Is it enough?

Comment (rleungx) [reply pkg/schedule/schedulers/balance_leader.go:89]: witness has handled this error, you can refer to it.

Comment (okJiang) [reply pkg/schedule/schedulers/balance_leader.go]: thanks for your suggestion. https://github.com/tikv/pd/pull/8346/commits/9de11548055277b10ec7dc329da86788eb114b95

**Review by rleungx — APPROVED** [review]

**Pull request merged**